go.mod

```diff
@@ -551,16 +564,16 @@ replace (
 	github.com/metal3-io/baremetal-operator/pkg/hardwareutils => github.com/openshift/baremetal-operator/pkg/hardwareutils v0.0.0-20221202075057-bc3fe35f8b62 // Use OpenShift fork
 	github.com/metal3-io/cluster-api-provider-baremetal => github.com/openshift/cluster-api-provider-baremetal v0.0.0-20190821174549-a2a477909c1d // Pin OpenShift fork
 	github.com/mholt/certmagic => github.com/caddyserver/certmagic v0.15.0
-	github.com/openshift/api => github.com/openshift/api v0.0.0-20230213134911-7ba313770556
+	github.com/openshift/api => github.com/openshift/api v0.0.0-20240103200955-7ca3a4634e46
```

Comment:
This version of openshift/api seems different to the version pinned in the fork. Is this coming from Hive: of https://github.com/openshift/installer-aro/blob/release-4.12-azure/go.mod#L263

Reply:
This was updated by `update-go-module-dependencies.sh`. this version here points to the newest commit on on the `release-4.12` branch of  openshift/api: https://github.com/openshift/api/commits/release-4.12/

Should I downgrade this again so it's aligned with the installer fork?

Reply:
I see, in that case if it's the latest of 4.12 and your tests already passed, it's ok to leave it.